src/main/webapp/scripts/map.js

```diff
@@ -77,10 +71,8 @@ function addMarker(map) {
 /* Place Marker Where Map is Clicked On & Show Popup*/
 function placeMarkerAndPanTo(latLng, map) {
   const greyIcon = {
```

Comment:
Can you please clarify the error you're getting if you tried to have the `geryIcon` as a global constant?

Reply:
Of course!

Here's the error:
![Screenshot 2020-08-26 at 12 17 39 PM - Display 2](https://user-images.githubusercontent.com/43929997/91297544-6b540300-e796-11ea-9992-5c369958120a.png)

Reply:
Thanks Lorna! can you please share the code snippet of the `map.js` file to better debug the error?

Reply:
![image](https://user-images.githubusercontent.com/43929997/91302333-3b106280-e79e-11ea-83c3-bf11625b8459.png)

Reply:
Thanks Lorna! I tried to debug the error but still don't know what's going wrong here, one thing you might try is to see if the `greyIcon` variable is visible inside the `placeMarkerAndPanTo` function by simply logging it, otherwise please feel free to have it as a local variable.

Reply:
I tried but can't work out what's wrong, so I'm keeping it as a local variable until after the merge (to free Rania from being blocked for the time being). I'll note this down and look into it though!